projects/js-packages/connection/components/connect-button/index.jsx

```diff
@@ -100,9 +100,9 @@ const ConnectButton = props => {
 
 					setAuthorizationUrl( response.authorizeUrl );
 					setIsUserConnecting( true );
-					setConnectionStatus( status => {
-						return { ...status, isRegistered: true };
-					} );
+					// setConnectionStatus( status => {
```

Comment:
Do we need to leave these comments here?

Reply:
The reason I commented out this code is that informing the parent about the site being registered triggers re-rendering, aka momentarily changing the UI while waiting for the redirect to happen.
Since the way this component is structured doesn't prevent the redirect (to establish user connection) from happening in any case I didn't see any value in this callback.
However I'd appreciate @sergeymitr 's take on this (who originally created that component) before we completely remove it.